packages/react-components/react-avatar/README-AvatarGroup.md

```diff
@@ -1,39 +1,39 @@
-# @fluentui/react-avatar
-
-**React AvatarGroup component for [Fluent UI](https://dev.microsoft.com/fluentui)**
-
-The AvatarGroup component represents a group of multiple people or entities by taking care of the arrangement of individual Avatars in a spread, stack, or pie layout.
-
-## STATUS: WIP 🚧
-
-These are not production-ready components and **should never be used in product**. This space is useful for testing new components whose APIs might change before final release.
-
-## Usage
-
-To import AvatarGroup and AvatarGroupItem:
-
-```js
-import { AvatarGroup, AvatarGroupItem } from '@fluentui/react-avatar';
-```
-
-Once the AvatarGroup component graduates to a production release, the component will be available at:
-
-```js
-import { AvatarGroup, AvatarGroupItem } from '@fluentui/react-components';
-```
-
-### Examples
-
-```jsx
-<AvatarGroup>
-  <AvatarGroupItem name="Katri Athokas" />
-  <AvatarGroupItem name="Elvia Atkins" />
-  <AvatarGroupItem name="Cameron Evans" />
-  <AvatarGroupItem name="Wanda Howard" />
-  <AvatarGroupItem name="Mona Kane" />
-  <AvatarGroupItem name="Allan Munger" />
-  <AvatarGroupItem name="Daisy Phillips" />
-  <AvatarGroupItem name="Robert Tolbert" />
-  <AvatarGroupItem name="Kevin Sturgis" />
-</AvatarGroup>
-```
+# @fluentui/react-avatar
```

Comment:
Do you know what changed in this file? Whitespace maybe?

Reply:
@rohitpagariya do you mind restoring these AvatarGroup files?

Reply:
No it's not. It's a whitespace change & I can't seem to undo it unfortunately.

Reply:
You can checkout from master, then commit with the flag --no-verify and that should work

Reply:
Nope, didn't work. Deleting & restoring didn't work either. I can even tell what's the diff tbh.

Reply:
Mind giving a try?

Reply:
done! if this problem comes up again you can do this from root:
`git checkout upstream/master packages/react-components/react-avatar/* && git commit --no-verify -m "{your message}"` and that should restore it to the original file.